yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -91,10 +97,22 @@ func runImportDataStatusCmd() error {
 		row := &tableMigStatusOutputRow{
 			tableName:          tableName,
 			status:             status,
-			totalRowCount:      totalRowCount,
-			importedRowCount:   importedRowCount,
+			totalRowCount:      float64(totalRowCount),
+			importedRowCount:   float64(importedRowCount),
 			percentageComplete: perc,
 		}
+
+		if dataFileDescriptor.TableRowCount == nil {
+			// case of importDataFileCommand where file size is available not row counts
+
+			// converting size in bytes to megabytes
+			totalRowSize := float64(totalRowCount) / 1000000.0
+			importedRowSize := float64(importedRowCount) / 1000000.0
+
+			row.totalRowCount = totalRowSize
+			row.importedRowCount = importedRowSize
```

Comment:
Perform this calculation right in the `Printf()` call.

Reply:
Done all of the above changes